Login_Forms/Glassmorphism_Form/code/style.css

```diff
@@ -0,0 +1,190 @@
+*{
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+body{
+	overflow: hidden;
+}
+section{

```

Comment:
Avoid styling html tags directly. Use classes instead

Reply:
Ok but as I am giving it to the whole body so I thought using tag is better.

Reply:
So the purpose of Dezenix is to provide templates for other people to use. With that in mind it is very probably that direct styling to html tags will conflict with the end-users existing html. So to avoid that we suggest using classes.

Reply:
> So the purpose of Dezenix is to provide templates for other people to use. With that in mind it is very probably that direct styling to html tags will conflict with the end-users existing html. So to avoid that we suggest using classes.

Ok will do that